src/sordid-dragon.jquery.coffee

```diff
@@ -135,31 +137,37 @@ do ($=jQuery) ->
         # Firefox won't trigger "drag" events without this.
         e.originalEvent.dataTransfer?.setData "text", ""
         $placeholder = $child.clone()
+        isDragging = true
 
       $handle.on "touchmove.sordidDragon drag.sordidDragon", (e) ->
-        showPlaceholder $child
-        if isTouch(e)
-          pageY = e.originalEvent.targetTouches[0].pageY
-          showGhost pageY
-
-          # On touch devices, we don't have dragenter events, so we'll update
-          # the position of the element being dragged here instead.
-          newPosition = currentPosition(pageY)
-          if newPosition?
-            moveChild $parent.children("#{options.childSelector || ""}:visible").eq(newPosition)
-          e.preventDefault()
+        if isDragging
```

Comment:
It can happen when sordidDragon is re-enabled during a drag. The old handlers can still be firing and get stuck. Shown in the new example https://github.com/polleverywhere/sordid-dragon/blob/2.0.0/examples/re_enable_basic.html#L26